log/log.go

```diff
@@ -3,52 +3,53 @@
 package log
 
 import (
-	"fmt"
+	"go.uber.org/zap/zapcore"
+	"gopkg.in/natefinch/lumberjack.v2"
+	"io"
 	"os"
 	"path/filepath"
 
-	"gopkg.in/natefinch/lumberjack.v2"
-	"gopkg.in/op/go-logging.v1"
+	"go.uber.org/zap"
 )
 
-// Log is an exported type that embeds our logger.
-// logging library can be replaced as long as it implements same functionality used across the project.
-type Log struct {
-	*logging.Logger
-}
 
-// smlogger is the local app singleton logger.
-var AppLog Log
+// determine the level of messages we show.
 var debugMode = false
+// should we format out logs in json
+const JSON = false // todo: configure
```

Comment:
make it configurable as part of this PR, otherwise we will have to keep re-building before testing it externally